css/src/components/table.scss

```diff
@@ -0,0 +1,67 @@
+$table-cell-padding-xxs: $spacer-4 !default;
+$table-cell-padding-xs: $spacer-5 !default;
+
+$table-caption-size: $font-size-9 !default;
+$table-caption-spacing: 0.125rem !default;
```

Comment:
`$table-caption-letter-spacing` - should this be referencing another letter spacing token?

We have `$letter-spacing-wide` in our typography tokens, but the value is too large. Should we introduce `$letter-spacing-medium`?

Reply:
Updated